package.json

```diff
@@ -97,6 +97,7 @@
     "webpack-merge": "^4.2.2"
   },
   "dependencies": {
+    "@json-editor/json-editor": "^2.9.0-beta.0",
```

Comment:
Makes no sense to depend on itself. Also there's no package update needed (no changed .lock file)